platform/ui/src/components/userPreferencesForm/HotKeysPreferences.js

```diff
@@ -10,7 +10,13 @@ import PropTypes from 'prop-types';
 
 export class HotKeysPreferences extends Component {
   static propTypes = {
-    hotkeyDefinitions: PropTypes.array.isRequired,
+    hotkeyDefinitions: PropTypes.arrayOf(
+      PropTypes.shape({
+        commandName: PropTypes.string,
+        keys: PropTypes.arrayOf(PropTypes.string),
+        label: PropTypes.string,
+      })
+    ).isRequired,
```

Comment:
Just curious, what happens if `hotkeyDefinitions` is an empty array?

Reply:
The component will be rendered without any hotkey field. Do you think it's worth to conditionally render an informative text that no hotkey defs were found?

Reply:
I think something like:

> No hotkeys are configured for this application. Hotkeys can be configured in the application's `app-config.js` file.

Should be good enough for now. We can think of something better as a future enhancement.

Reply:
Im using the message directly for now, I have added the key/value to locize as well but I'm not sure about locals update.

Reply:
If it works locally, and it works in the deploy preview, then I think we're fine.